docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
+- Travis Build runs without errors (green)
+- Meaningful javadoc comments for:
+  - All classes
+  - All non-private methods and fields
+  - Private methods with non-obvious implementation
```

Comment:
We should add that for trivial methods (like getters and setters) javadocs can be omitted. I rather have no javadoc than something like getX(): 'returns the x'.
Regarding 'Private methods with non-obvious implementation': Explanation of implemention details do not belong into javadocs. they have to be written as normal comments inside the methods. Remember, javadoc is mainly for the consumer of the code.